spec/specialist_publisher_spec.rb

```diff
@@ -5,7 +5,7 @@
 
 describe "specialist publisher", type: :feature do
   feature "Publishes an AAIB Report" do
-    let(:title) { Faker::Book.author }
+    let(:title) { Faker::Book.author << Time.now.to_s }
```

Comment:
I imagine this comes out a big ugly?

```
irb(main):006:0> "Test" << Time.now.to_s
=> "Test2017-03-06 09:58:00 +0000"
```

it'd be good to get this a little smaller. Options I'd consider:
```
irb(main):004:0> "Test now" << " #{Time.now.to_i}"
=> "Test now 1488794186"
irb(main):005:0> "Test now" << " #{Time.now.utc.iso8601}"
=> "Test now 2017-03-06T09:57:16Z"
```

the former being my preference as it's briefest.

Also I think this would make sense being part of a method call - maybe `append_timestamp(Faker::Book.author)`  - which makes it's usage clearer?